src/Helpers/Consign/index.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * Helpers for /consign app.
+ *
+ * @see https://docs.google.com/spreadsheets/d/12wj1rt-j5SguboPJMuaM1GqJD-5B5u5piipw8MQ5Zuk/edit#gid=0 for matrix.
+ */
+
+import {
+  ActionType,
+  ClickedAppDownload,
+  ClickedArticleGroup,
+  ClickedArtworkGroup,
+  ContextModule,
+  OwnerType,
+} from "../../Schema"
+import {
+  FocusedOnSearchInput,
+  SearchedWithNoResults,
+  SelectedItemFromSearch,
+} from "../../Schema/Events/Search"
+
+type ExcludeDefined<T> = Required<
+  Omit<
+    T,
+    "action" | "context_module" | "context_owner_type" | "type" | "subject"
+  >
+>
+
+export const focusedOnSearchInput = (
+  props: ExcludeDefined<FocusedOnSearchInput>,
```

Comment:
Search events should live together in a `Helpers/Search` directory.

Reply:
Cool, so in that case we pass in below hard-coded values as props and this function becomes reusable 👍